### PR TITLE
Add welcome page

### PR DESCRIPTION
**Pull request by alexandruradovici**
### Pull Request Overview

This pull request adds the welcome page with:
- the team
- the rules
- the timetable
- the grading
- the bibliography

### Testing Strategy

N/A

### TODO or Help Wanted

This pull request still needs feedback. Please verify that your details and photo are right.

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Alexandru Radovici <alexandru.radovici@wyliodrin.com>

### CODE REVIEW

File: docs/welcome/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+| 1222A | Thursday | 14 - 16 | ED 218 |
+| 1222B | Thursday | 12 - 14 | ED 218 |
+
+## Grading
+
+| Part | Description | Points |
+|--------|-------------|--------|
+| [Lecture](/docs/category/lecture) tests | You will have a test at every class with subjects from the previous class. | 2p |
+| [Lab](/docs/category/lab) | Your work at every lab will be graded. | 2p |
+| [Project](/docs/project) | You will have to design and implement a hardware device. Grading will be done for the documentation, hardware design and software development. | 5p |
```
Comment (DTeodor-Alexaandru) [inline docs/welcome/index.md:81]: + Also PM fair
Proposal: each milestone completed 1p and PM Fair 2p

**Review by ZamfirAndreiPaul — DISMISSED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Comment by alexandruradovici** [comment]
This will be merge only when all of you approve it.

File: docs/welcome/index.md (hunk 1 above)
Comment (alexandruradovici) [reply docs/welcome/index.md:83]: My idea is to add those in the project description.

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Comment by alexandruradovici** [comment]
@ZamfirAndreiPaul @irina-b-dev @irina-nita @DanutAldea review please.

**Review by ZamfirAndreiPaul — DISMISSED** [review]

**Review by irina-nita — APPROVED** [review]

File: docs/welcome/index.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+---
+
+# Welcome
+
+Welcome to the *Microprocessor Architecture* class.
+
+During the class you will learn how to design and implement a hardware device. 
+
+You takeaways from the course are:
+  - learning how a microporcessor (*MCU*) works, using as an example the RP2040 MCU from the [Raspberry Pi Pico W](https://www.raspberrypi.com/products/raspberry-pi-pico/);
```
Comment (irina-nita) [inline docs/welcome/index.md:13]:
```suggestion
  - learning how a microprocessor (*MCU*) works, using as an example the RP2040 MCU from the [Raspberry Pi Pico W](https://www.raspberrypi.com/products/raspberry-pi-pico/);
```

**Comment by alexandruradovici** [comment]
@ZamfirAndreiPaul @irina-nita @DTeodor-Alexaandru @DanutAldea @irina-b-dev if you want to user you university email accounts, please edit your email using `suggest changes` in this pull request.

**Review by DTeodor-Alexaandru — DISMISSED** [review]

File: docs/welcome/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+</tr>
+
+<tr valign="top">
+<td>
+**Dănuț Aldea**
+![Danut Aldea](images/danut_aldea.jpg)
+ 
+Lab Professor \
+GitHub: [danutaldea](https://github.com/danutaldea) \
+danut.aldeaa@oxidos.io
```
Comment (DanutAldea) [inline docs/welcome/index.md:59]:
```suggestion
george_danut.aldea@stud.acs.upb.ro
```

**Review by DanutAldea — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Last call for this @DanutAldea @ZamfirAndreiPaul @irina-nita @irina-b-dev @DTeodor-Alexaandru

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Review by DanutAldea — APPROVED** [review]

**Pull request merged**